### ISSUE TITLE
Posts with private status get lost

### ISSUE THREAD

**Issue by shawon786** [tareq1988/wedocs#54]
If anyone does not want front-end users to see any post and just set the status to '**private**', it gets disappeared. 
Even admin can't see private posts under weDocs.

**Comment by coolmanlume** [comment]
Indeed, same Problem here. Both Admins and Editors can't see/read the Docs and/or Sections once set to private (**vs [standard WP behavior](https://wordpress.org/support/article/content-visibility/#private-content)**). They can still edit it, but to access the private content they need to search or use other circumvolutions.
